dev/make-distribution.sh

```diff
@@ -163,7 +163,7 @@ echo -e "\$ ${BUILD_COMMAND[@]}\n"
 rm -rf "$DISTDIR"
 mkdir -p "$DISTDIR/jars"
 echo "Spark $VERSION$GITREVSTRING built for Hadoop $SPARK_HADOOP_VERSION" > "$DISTDIR/RELEASE"
-echo "Build flags: $@" >> "$DISTDIR/RELEASE"
+echo "Build flags: "$@"" >> "$DISTDIR/RELEASE"
```

Comment:
I think this is okay since it's just printing out some message, in fact I think the original command is also valid and looks better.